src/isotools/_utils.py

```diff
@@ -316,28 +333,28 @@ def _get_overlap(exons, transcripts):
     if not transcripts:
         return 0
     # 1) get exononic regions in transcripts
-    exon_reg = _get_exonic_region(transcripts)
+    exon_region = _get_exonic_region(transcripts)
     # 2) find overlap of exonic regions with exons
     ol = 0
     i = 0
-    for e in exons:
-        while exon_reg[i][1] < e[0]:  # no overlap, go on
+    for exon in exons:
+        while exon_region[i][1] < exon[0]:  # no overlap, go on
             i += 1
-            if i == len(exon_reg):
+            if i == len(exon_region):
                 return ol
-        while exon_reg[i][0] < e[1]:
-            i_end = min(e[1], exon_reg[i][1])
-            i_start = max(e[0], exon_reg[i][0])
+        while exon_region[i][0] < exon[1]:
+            i_end = min(exon[1], exon_region[i][1])
+            i_start = max(exon[0], exon_region[i][0])
             ol += (i_end - i_start)
-            if exon_reg[i][1] > e[1]:  # might overlap with next exon
+            if exon_region[i][1] > exon[1]:  # might overlap with next exon
                 break
             i += 1
-            if i == len(exon_reg):
+            if i == len(exon_region):
                 return ol
     return ol
 
 
-def _find_splice_sites(sj, transcripts):
+def _find_splice_sites(splice_junctions, transcripts):
```

Comment:
In this function, I think it's better to expand 'sjs' to 'splice_junction_start'. Then also expand 'sje' to 'splice_junction_end' since you did this for 'sjs'.